docs/LoadCellProbe.md

```diff
@@ -0,0 +1,288 @@
+# Load-cell based probe
+
+A load-cell based probe offers an offset-free measurement of the precise
+Z coordinate of the contact position between nozzle and print bed. Under certain
+mechanical conditions, a sensitivity can be even achieved in X and Y directions,
+which can be useful for CNC routers. Downside of a load-cell probe compared to
+other probe types is the slower speed and higher complexity.
+
+Please also read the
+[reference of the config section](Config_Reference.md#load_cell_probe), and the
+[reference of the g-code commands](G-Codes.md#load_cell_probe).
+
+## Mechanical and electrical setup
+
+<p align="center">
+  <img src="img/load-cell-probe-setup.png" width="30%"/>
+</p>
+
+There may be many possibilities how to realise a probe based on load cells. The
+setup used for testing and developing this algorithm is as follows:
```

Comment:
FWIW, I think the document would be easier for users if it started with a concrete set of steps necessary to calibrate and use their probe.  The theory sections are helpful, but I'd put them towards the end of the document.  In my experience, the majority of users just want to follow a guide with the steps to follow and are uninterested in obscure alternatives or theory